File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count = Counter(nums)
        arr = []
        for key,val in count.items():
            arr.append([val, key])
        arr.sort()

        final = []
        while len(final) < k:
            final.append(arr.pop()[1])
        return final

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] ans = new int[2*nums.length];
        int k=0;
        for(int i=0;i<ans.length;i++){
            ans[i]=nums[k]; 
            k++;
            if(k>=nums.length){
             k=0;
            }
            
            
        }
        return ans;
    }
}